File: Program25.java
import java.lang.*;
import java.util.*;

class Number
{
    public int iNo;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number");
        this.iNo = sobj.nextInt();
    }

    public void Display()
    {
        System.out.println("Value is: "+this.iNo);
    }

    
   public boolean Check()
   {
       if(
           ((iNo % 3) == 0) &&
           ((iNo % 5) == 0)
         )
        {
            return true; 
        }
        else
        {
            return false;
        }
    }
}

class Program25
{
    public static void main(String arg[])
    {
        boolean iRet ;

        Number nobj = new Number();

        nobj.Accept();
        nobj.Display();
        
        iRet = nobj.Check();
        if(iRet == true)
        {
          System.out.print("Divisible by 3 and 5");
        }
        else
        {
          System.out.print("Not Divisible by 3 and 5");
        }
    }
}

File: Program25.cpp
#include<iostream>
using namespace std;

class Number
{
    public:
       int iNo;
    
    public:
       void Accept()
       {
          cout<<"Enter number"<<endl;
          cin>>this->iNo;
       }

       void Display()
       {
           cout<<"Value is: "<<this->iNo<<endl;
       }

       bool Check()
       {
           if(
               ((iNo % 3) == 0) &&
               ((iNo % 5) == 0)
             )
           {
               return true;
           }
           else
           {
               return false;
           }
       }
};

int main()
{
    int iRet;

    Number nobj;

    nobj.Accept();
    nobj.Display();

    iRet = nobj.Check();
    
    if(iRet == true)
    {
       cout<<"Divisible by 3 & 5";
    }
    else
    {
        cout<<"Not Divisible by 3 & 5";
    }
    
    return 0;
}